docs/manual/customization.md

```diff
@@ -247,23 +247,23 @@ Our [sample deployment](https://raw.githubusercontent.com/foundationdb/fdb-kuber
 
 ### Single-Namespace Mode
 
-To use single-namespace mode, set the `WATCH_NAMESPACE` environment variable for the controller to be the namespace where your FDB clusters will run. It does not have to be the same namespace where the operator is running, though this is generally the simplest way to configure it. When you are running in single-namespace mode, the controller will ignore any clusters you try to create in namespaces other than the one you give it.
+To use single-namespace mode, set the `WATCH_NAMESPACE` environment variable or the command-line option `-watch-namespace` for the controller to be the namespace where your FDB clusters will run. It does not have to be the same namespace where the operator is running, though this is generally the simplest way to configure it. When you are running in single-namespace mode, the controller will ignore any clusters you try to create in namespaces other than the one you give it. If both options are defined, the environment variable will take precedence.
```

Comment:
I would remove the env variable option in the docs and only mention the new cli flag.

Reply:
The environment variable is useful because you can set it to specify the namespace the operator is running in via the downward API, which AFAIK is not possible for command-line arguments.

Reply:
That makes sense and is a valid concern. One option would be to have the env variable `WATCH_NAMESPACE` as the default value for the `--watch-namespace` flag? That would support both the usage of the env variable and the specific passing of the new cli flag?

Reply:
That makes sense to me.